mapApp/templates/mapApp/incident_form.html

```diff
@@ -131,13 +131,13 @@ <h4 class="modal-title" id="incidentForm">{% trans "Submit a new report" %}</h4>
       $('#message .alert').alert('close');
     }, 7000);
   };
-  
-  function showHazardMessage(message){
+
+  function showFollowUpInfo(message){
     $('#311popUp').append('<div id="popUpMessage">' +

```

Comment:
Would you mind renaming the '311popup' to something more generic? The 311 is pretty specific to the Vancouver geofence message I think.